caniusepython3/overrides.json

```diff
@@ -35,12 +37,15 @@
     "pypdf": "https://pypi.python.org/pypi/PyPDF2",
     "pyrss2gen": "https://pypi.python.org/pypi/PyRSS2Gen",
     "pysqlite": "http://docs.python.org/3/library/sqlite3.html",
+    "python-ldap": "https://pypi.python.org/pypi/pyldap",
```

Comment:
Agreed.  The trove classifiers are in [pyldap's setup.py file](https://github.com/pyldap/pyldap/blob/master/setup.py#L120) but not in the PyPI record.  See https://github.com/pyldap/pyldap/pull/52